### ISSUE TITLE
predis/predis doesn't support replication false

### ISSUE THREAD

**Issue by viniciusss** [snc/SncRedisBundle#683]
I'm trying to use version 4.4.0 and i'm receiving the error "Values evaluating to FALSE are not accepted for `replication`" .

This is my redis.yaml config:

```yaml
snc_redis:
    clients:
        default:
            type: predis
            alias: default
            dsn: '%env(REDIS_URL)%'
```

To resolve the problem i returned to the 4.3.2 version.

## Ref

Link to predis error: https://github.com/predis/predis/commit/ca468b785c60c1a342f8cdd3895dc427bdbad012

The commit who caused the error https://github.com/snc/SncRedisBundle/commit/6f654e43ed9042e4ca53496e7a2b109fc6749ae0#diff-4e41bb7ece752048ea9a68d151a55920b7613e2e9430ef7714db42cf0933589cR129

**Comment by alex-bacart** [comment]
Having the same issue and also downgraded to 4.3.2

**Comment by ostrolucky** [comment]
Thanks for quick report, this should be fixed in 4.4.1
